src/controllers/workshiftController.js

```diff
@@ -61,6 +87,13 @@ export const createWorkshiftsBulk = async (req, res) => {
     const { doctorId, clinicId, duration, periodStartDate, periodEndDate } = req.body;
 
     if (!doctorId || !clinicId || !periodStartDate || !periodEndDate) {
+      logger.error('Error creating workshifts: doctorId, clinicId, periodStartDate, and periodEndDate are required', {
+        method: req.method,
+        url: req.originalUrl,
+        body: req.body,
```

Comment:
`body` should not appear in the logs due to privacy policy terms, the format on errors should be:
 ```json
      method: req.method,
      url: req.originalUrl,
      error: error.message,
      ip: req.headers && req.headers['x-forwarded-for'] || req.ip,
      requestId: req.headers && req.headers['x-request-id'] || null,
```